ui/src/logs/actions/index.ts

```diff
@@ -38,11 +47,36 @@ interface SetTimeRangeAction {
   }
 }
 
+interface SetHistogramQueryConfig {
+  type: ActionTypes.SetHistogramQueryConfig
+  payload: {
+    queryConfig: QueryConfig
+  }
+}
+
+interface SetHistogramData {
+  type: ActionTypes.SetHistogramData
+  payload: {
+    data: object[]
```

Comment:
anything more specific we could expect here?

Reply:
As this is a WIP feature some of the types are in flux, but there we be a concrete type forthcoming.

Reply:
👍